### PR TITLE
Add university of oslo

### PR DESCRIPTION
**Pull request by cserf**

### CODE REVIEW

File: _gsocorgs/2018/uoslo.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ---
 title: "University of Oslo"
-author: "Mario Lassnig"
+author: "Cédric Serfon"
 layout: default
-organization: UOslo
+organization: UiO
+logo: University-of-Oslo.png
```
Comment (etejedor) [inline _gsocorgs/2018/uoslo.md]: Hi Cedric, thanks for the changes.
I noticed there is already a logo image of UOslo in the repo:
https://github.com/HEP-SF/hep-sf.github.io/blob/master/images/UOslo.png

If that image is ok for you, I would propose to remove the logo from the PR and refer to UOslo.png from your uoslo.md file.

File: _gsocorgs/2018/uoslo.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 ---
 title: "University of Oslo"
-author: "Mario Lassnig"
+author: "Cédric Serfon"
 layout: default
-organization: UOslo
```
Comment (etejedor) [inline _gsocorgs/2018/uoslo.md]: Please note that if you change the identifier of the university from UOslo to UiO, your proposal:
https://raw.githubusercontent.com/HEP-SF/hep-sf.github.io/master/_gsocproposals/2018/proposal_ATLASrucioMetadata.md
will need to point to that new identifier.

Comment (cserf) [reply _gsocorgs/2018/uoslo.md]: I change to the new identifier

File: _gsocorgs/2018/uoslo.md (hunk 1 above)
Comment (cserf) [reply _gsocorgs/2018/uoslo.md]: I think the one you mention is an old one.

**Pull request merged**